package.json

```diff
@@ -44,10 +46,13 @@
     "fs": "0.0.1-security",
     "helmet": "^4.6.0",
     "jsonwebtoken": "^8.5.1",
+    "jszip": "^3.7.1",
     "jwks-rsa": "^2.0.5",
     "jwt-decode": "^3.1.2",
     "lodash": "^4.17.21",
+    "moment": "^2.29.1",
     "morgan": "^1.10.0",
+    "node-fetch": "^2.6.6",
```

Comment:
lol out of sheer curiosity: what's wrong with axios?

Reply:
i just preferred using fetch here

Reply:
I asked because, for consistency, I believe we should use a single library for requests.
Mind if I ask you change this? that way we can avoid having to remove it from other existing endpoints.

As a side note: my preference for Axios is that it offers `cancellation` which may come in handy later, and because it allows creating reusable instances/configs. If `Node-Fetch` offers those, I am all in for migrating.